packages/airnode-node/src/constants.test.ts

```diff
@@ -1,25 +1,12 @@
-import * as constants from './constants';
+import {
+  WORKER_CALL_API_TIMEOUT,
+  WORKER_PROCESS_TRANSACTIONS_TIMEOUT,
+  WORKER_PROVIDER_INITIALIZATION_TIMEOUT,
+} from './constants';
 
-describe('constants', () => {
-  it('returns constant values', () => {
-    const constCount = Object.keys(constants).length;
-
-    // Ensure all constant values are checked
-    expect.assertions(constCount + 1);
-    expect(Object.keys(constants).length).toEqual(13);
-
-    expect(constants.API_CALL_TIMEOUT).toEqual(30_000);
-    expect(constants.API_CALL_TOTAL_TIMEOUT).toEqual(30_000);
-    expect(constants.BLOCK_COUNT_HISTORY_LIMIT).toEqual(300);
-    expect(constants.BLOCK_MIN_CONFIRMATIONS).toEqual(0);
-    expect(constants.CONVENIENCE_BATCH_SIZE).toEqual(10);
-    expect(constants.DEFAULT_RETRY_TIMEOUT_MS).toEqual(5_000);
-    expect(constants.EVM_PROVIDER_TIMEOUT).toEqual(10_000);
-    expect(constants.WORKER_CALL_API_TIMEOUT).toEqual(30_000);
-    expect(constants.WORKER_PROVIDER_INITIALIZATION_TIMEOUT).toEqual(20_000);
-    expect(constants.WORKER_PROCESS_TRANSACTIONS_TIMEOUT).toEqual(10_000);
-    expect(constants.MAXIMUM_SPONSOR_WALLET_REQUESTS).toBe(5);
-    expect(constants.MAXIMUM_ONCHAIN_ERROR_LENGTH).toEqual(100);
-    expect(constants.PROCESSING_TIMEOUT).toEqual(10_000);
-  });
+it('worker timeouts are smaller then terraform lambda timeout', () => {
+  // The 32 seconds timeout is currently hardcoded in TF
+  expect(WORKER_PROVIDER_INITIALIZATION_TIMEOUT).toBeLessThan(32_000);
```

Comment:
This will be true even for `31_999` but that won't leave the desired 2 seconds for a graceful termination. What about something like `constant <= TF value - grace period (2 sec)`?

Reply:
I think that should be done when we implement the 1. suggestion in https://github.com/api3dao/airnode/issues/217#issuecomment-1224141791.

Or do you want to add that grace period now? (I don't have a strong preference)

Reply:
No, that sounds fine.